src/types.ts

```diff
@@ -1,14 +1,18 @@
 import { BigNumber, ethers } from "ethers";
 
-export interface SubConfig {
+export interface FactoryConfig {
```

Comment:
Factory seems like the wrong word here

Reply:
There is a `Config` used to create an instance of the SDK, but that internally created a `SubConfig` that was used for each individual pool and factory. Now there is need to have pool configs be separate from the factory config, because the pool configs now require a subgraph URI while the factory configs don't. So a ~FactoryConfig` has props needed for both factories and pools, and a `PoolConfig` extends from `FactoryConfig` also including a subgraph URI. 

If you still feel that is not indicative please let me know what you'd like to have it changed to.